src/condor_utils/file_transfer.cpp

```diff
@@ -265,6 +265,8 @@ dPrintFileTransferList( int flags, const FileTransferList & list, const std::str
 			i.srcName().c_str(), i.destDir().c_str(), i.destUrl().c_str()
 		);
 	}
+	// Don't print the trailing comma.
+	message[message.size()] = '\0';
```

Comment:
Three problems with this line of code:
* It doesn't do anything (you wanted size()-1)
* You should be shortening the string (e.g. with erase()), not overwriting characters with NUL.
* If the list is empty, you erase part of the header string.

Reply:
Strictly speaking, none of this code does anything because all of the calls are commented out, but fine. :)